bin/configs/avro-schema-issue6268.yaml

```diff
@@ -0,0 +1,4 @@
+generatorName: avro-schema
+outputDir: samples/openapi3/schema/petstore/avro-schema-issue6268
```

Comment:
thanks for the PR. is there any command to validate the output?

what about adding a github workflow to do that?

ref: https://github.com/OpenAPITools/openapi-generator/tree/master/.github/workflows

Reply:
pr merged. we can improve tests later with separate PRs.